src/PhpSpec/CodeGenerator/Generator/PromptingGenerator.php

```diff
@@ -148,9 +159,23 @@ private function createDirectoryIfItDoesExist($filepath)
      */
     private function generateFileAndRenderTemplate(ResourceInterface $resource, $filepath)
     {
+        $fileExists = $this->fileAlreadyExists($filepath);
         $content = $this->renderTemplate($resource, $filepath);
 
         $this->filesystem->putFileContents($filepath, $content);
         $this->io->writeln($this->getGeneratedMessage($resource, $filepath));
+
+        if (!$fileExists) {
```

Comment:
What is this check guarding against? The file already existed but we generated it anyway?